addons/website/tests/test_snippets.py

```diff
@@ -79,17 +80,20 @@ def test_08_table_of_content(self):
     def test_09_snippet_image_gallery(self):
         IrAttachment = self.env['ir.attachment']
         base = "http://%s:%s" % (HOST, config['http_port'])
+        req = self.opener.get(base + '/web/image/website.s_banner_default_image')
         IrAttachment.create({
             'public': True,
             'name': 's_default_image.jpg',
-            'type': 'url',
-            'url': base + '/web/image/website.s_banner_default_image.jpg',
+            'type': 'binary',
+            'res_model': 'ir_ui_view',
+            'datas': base64.b64encode(req.content),
```

Comment:
> The two attachments needed for the snippet_image_gallery_remove test have slightly been adapted in order for the /web_editor/get_image_info route to correctly find them. (As since this version, we handle absolute url).

I am not sure to understand that change and that explanation, could you elaborate? :pray:

Reply:
> I am not sure to understand that change and that explanation, could you elaborate?

@qsm-odoo , Sure! With the attachment created as before the modification;
- In version before 16, the absolute URL (e.g. `http://127.0.0.1:8069/web/image/website.s_banner_default_image.jpg`) is given to the `/web_editor/get_image_info` route to retrieve the original attachment linked to the image. Thanks to it, the system manages to find this original attachment. The mimetype attribute of the image is filled and the options such as `Filter` (needed for the rest of the `snippet_image_gallery` tour) are displayed.
- From version 16, there is only the relative URL that is given to the `/web_editor/get_image_info` route to retrieve the original attachment. Due to that, the system fails to find the original attachment. The mimetype attribute is not added on the image and the option `Filter` is not available on the editor panel. Due to that, the tour fails when trying to change the filter of the new added image.

To solve the problem, I created the two needed attachments with a different method than by created them from their url's.

Reply:
> Due to that, the system fails to find the original attachment.

@loco-odoo It sounds like a bug that should be fixed instead of ignored, no? But I am not sure to understand everything yet. What does the test actually do? With or without URL type, when added in a page, they should be something like `/web/image/XXX` and then finding the original attachment should be about finding the XXX attachment whether it is or URL type or not?